books/RayTracingTheNextWeek.html

```diff
@@ -965,6 +965,71 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [hit-record-uv]: <kbd>[hittable.h]</kbd> Adding U,V coordinates to the `hit_record`]
 
+We will also need to compute $(u,v)$ texture coordinates for hittables.
+
+Texture Coordinates for Spheres
+--------------------------------
+For spheres, this is usually based on some form of longitude and latitude, _i.e._, spherical
```

Comment:
This needs to begin with a `<div class='together'>` to match the closing `</div>` on line 979.